src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/MiningAndroidListener.java

```diff
@@ -37,6 +41,10 @@ public void onAndroidMine(AndroidMineEvent e) {
             slimefunItem.callItemHandler(BlockBreakHandler.class, handler -> {
                 if (handler.isAndroidAllowed(e.getBlock())) {
                     handler.onAndroidBreak(e);
+
+                    Collection<ItemStack> drops = e.getDrops();
+                    drops.clear();
+                    drops.add(slimefunItem.getItem().clone());
```

Comment:
SlimefunItems have `getDrops` too, it should respect that... instead of just blindly cloning the default itemstack